src/components/UserPortal/UserNavbar/UserNavbar.tsx

```diff
@@ -20,24 +20,30 @@ function userNavbar(): JSX.Element {
     keyPrefix: 'userNavbar',
   });
 
-  const { getItem } = useLocalStorage();
-
   const [revokeRefreshToken] = useMutation(REVOKE_REFRESH_TOKEN);
 
   const [currentLanguageCode, setCurrentLanguageCode] = React.useState(
     /* istanbul ignore next */
     cookies.get('i18next') || 'en',
   );
 
+  const [userName, setUserName] = React.useState('');
+
+  const { data } = useQuery(CHECK_AUTH);
+
+  React.useEffect(() => {
+    if (data) {
+      setUserName(`${data.checkAuth.firstName} ${data.checkAuth.lastName}`);
```

Comment:
Shouldn't we set this in `localStorage` instead of react state?

Reply:
We had issues with using `localStorage` in the past. Our PR workflows will flag its use and fail because of this.

Reply:
I'm trying to fix this but facing the [above](https://github.com/PalisadoesFoundation/talawa-admin/pull/1533#issuecomment-1930625459) mentioned issue.